devicetypes/smartthings/smartsense-motion-sensor.src/smartsense-motion-sensor.groovy

```diff
@@ -133,7 +133,7 @@ def parse(String description) {
 						map = getBatteryResult(Integer.parseInt(battMap.value, 16))
 					}
 				}
-			} else if (descMap?.clusterInt == 0x0500 && descMap.attrInt == 0x0002) {
+			} else if (descMap?.clusterInt == 0x0500 && descMap.attrInt == 0x0002 && descMap.commandInt != 0x07)) {
```

Comment:
Is the NPE check not necessary on `attrInt` and commandInt` because they're guaranteed to exist?

Reply:
I originally had a null check, but I'm like 90% sure these are config reporting responses so I went back to this.

You're probably right.

Reply:
I've decided to stick with this because it seems to be a pattern employed elsewhere.